integrations/datadog/config.yaml

```diff
@@ -1,141 +1,69 @@
 ---
 jobs:
 - name: "materialize"
-  interval: '5m'
+  interval: '1m'
   connections:
   - "postgres://<USER>:<PASSWORD>@<HOST>:<PORT>/materialize?options=--cluster%3Dmz_introspection"
   queries:
-  - name: "source_messages_received"
-    help: "Count of messages for each source"
+  - name: "cluster_replica_usage"
+    help: "Cluster replica metrics"
     labels:
-      - "source_name"
-      - "source_type"
-      - "cluster_id"
-      - "cluster_name"
-    values:
-      - "messages_received"
-    query:  |
-            SELECT
-              SUM(messages_received) as messages_received,
-              S.name as source_name,
-              S.type as source_type,
-              S.cluster_id,
-              S.name as cluster_name
-            FROM mz_internal.mz_source_statistics SS
-            JOIN mz_catalog.mz_sources S ON (SS.id = S.id)
-            JOIN mz_catalog.mz_clusters C ON (S.cluster_id = C.id)
-            GROUP BY S.name, S.type, S.cluster_id, cluster_name;
-  - name: "source_bytes_received"
-    help: "Volume of messages for each source"
-    labels:
-      - "source_name"
-      - "source_type"
+      - "replica_id"
       - "cluster_id"
       - "cluster_name"
+      - "replica_name"
+      - "replica_status_label"
+      - "cluster_type"
     values:
-      - "bytes_received"
+      - "cpu_percent"
+      - "memory_percent"
+      - "credits_per_hour"
+      - "replica_status"
     query:  |
             SELECT
-              SUM(bytes_received) as bytes_received,
-              S.name as source_name,
-              S.type as source_type,
-              S.cluster_id,
-              S.name as cluster_name
-            FROM mz_internal.mz_source_statistics SS
-            JOIN mz_catalog.mz_sources S ON (SS.id = S.id)
-            JOIN mz_catalog.mz_clusters C ON (S.cluster_id = C.id)
-            GROUP BY S.name, S.type, S.cluster_id, cluster_name;
-  - name: "sink_messages_commited"
-    help: "Count of messages for each sink"
+              U.replica_id,
+              R.cluster_id,
+              C.name as cluster_name,
+              R.name as replica_name,
+              RST.status as replica_status_label,
```

Comment:
OOM's are surfaced in the `reason` column of `mz_internal.mz_cluster_replica_statuses`.

Reply:
Adding!